dump/python3.15/index.py

```
@@ -0,0 +1,22 @@
+import os
+import sys
+import subprocess
+import json
+
+def handler(event, context):
+    bucket = os.environ['BUCKET']
+    subprocess.call(['sh', '-c', f'lambda-dump -bucket {bucket} -key fs/__ARCH__/python3.15.tgz'])
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)